src/components/v5/common/ActionSidebar/partials/motions/MotionSimplePayment/MotionSimplePayment.tsx

```diff
@@ -1,72 +1,104 @@
 import { formatText } from '~utils/intl';
+import { useGetColonyAction } from '~v5/common/ActionSidebar/hooks/useGetColonyAction';
 import Stepper from '~v5/shared/Stepper';
+
+import MotionCountDownTimer from './partials/MotionCountDownTimer';
 import FinalizeStep from './steps/FinalizeStep';
 import OutcomeStep from './steps/OutcomeStep';
 import RevealStep from './steps/RevealStep';
 import StakingStep from './steps/StakingStep';
 import VotingStep from './steps/VotingStep';
+import { MotionSimplePaymentProps } from './types';
 
 const displayName =
   'v5.common.ActionSidebar.partials.motions.MotionSimplePayment';
 
-const MotionSimplePayment: FC = () => {
+const MotionSimplePayment: FC<MotionSimplePaymentProps> = ({
+  transactionId,
+}) => {
+  const { action, motionState, refetchMotionState, loadingAction } =
+    useGetColonyAction(transactionId);
+  const { motionData } = action || {};
+  const { motionId = '', motionStakes } = motionData || {};
   // @todo: pass current step to the state when other steps will be available
-  const [activeStepIndex, setActiveStepIndex] = useState(0);
+  const [activeStepIndex, setActiveStepIndex] = useState(motionState || 0);
 
   const items = useMemo(
-    () => [
-      {
-        key: '1',
-        content: <StakingStep />,
-        heading: {
-          label: formatText({ id: 'motion.staking.label' }) || '',
-        },
-      },
-      {
-        key: '2',
-        content: <VotingStep />,
-        heading: {
-          label: formatText({ id: 'motion.voting.label' }) || '',
-        },
-        // @todo: add a condition to be required if staking won't go directly to finalize step
-        isOptional: true,
-      },
-      {
-        key: '2.5',
-        content: <RevealStep />,
-        heading: {
-          label: formatText({ id: 'motion.reveal.label' }) || '',
-        },
-        // @todo: add a condition to show when voting step is active
-        isHidden: true,
-        // @todo: chnage to false when visible
-        isOptional: true,
-      },
-      {
-        key: '3',
-        content: <OutcomeStep />,
-        heading: {
-          // @todo: chnage label and styling when the outcome is known and revealed
-          label: formatText({ id: 'motion.outcome.label' }) || '',
-        },
-        // @todo: add a condition to be required if staking won't go directly to finalize step
-        isOptional: true,
-        // @todo: add a condition to hide when voting step is skipped
-        isHidden: false,
-      },
-      {
-        key: '4',
-        content: <FinalizeStep />,
-        heading: {
-          label: formatText({ id: 'motion.finalize.label' }) || '',
-        },
-      },
-    ],
-    [],
+    () =>
+      loadingAction
+        ? []
+        : [
+            {
+              key: '1',
+              content: <StakingStep />,
+              heading: {
+                label: formatText({ id: 'motion.staking.label' }) || '',
```

Comment:
why formatText or empty string?

Reply:
Label is required and formatText returns string or undefined